python-app/openapi-diff/action.yaml

```diff
@@ -0,0 +1,98 @@
+name: OpenApi Diff
+description: Generate and comment OpenAPI Diff
+
+inputs:
+  base_ref:
+    description: "Base branch for the diff"
+    required: true
+    default: "main"
+  head_ref:
+    description: "Head branch for the diff"
+    required: true
+
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: set up python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.11"
```

Comment:
Maybe you can add a cache key here so we gain time by not re downloading all libraries over and over again.
Documentation says:
```
- uses: actions/setup-python@v4
  with:
    python-version: '3.9'
    cache: 'pipenv' # caching pipenv dependencies
```

Reply:
Yes indeed, agreed